### PR TITLE
🐛 PRTL-1294 fix login popup not closing

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: src/packages/@ncigdc/components/LoginButton.js [hunk 1]
```diff
@@ -17,13 +13,7 @@ import { AUTH } from '@ncigdc/utils/constants';
...
 const openAuthWindow = ({ pathname, dispatch }) => {
   if (navigator.cookieEnabled) {
-    const returningPath = `${pathname}?${+new Date()}`;
-    const redirectUrl = urlJoin(AUTH, `?next=${returningPath}`);
-
-    const closeLogin = url =>
-      url === redirectUrl ? false : url.includes(returningPath);
-
-    const win = open(redirectUrl, 'Auth', 'width=800, height=600');
+    const win = open(AUTH, 'Auth', 'width=800, height=600');
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/LoginButton.js]: AUTH is not a global need to import from constants

Comment (shanewilson) [inline src/packages/@ncigdc/components/LoginButton.js]: oh maybe just need to rebase?

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/LoginButton.js]: line 10?

Comment (shanewilson) [inline src/packages/@ncigdc/components/LoginButton.js]: oh it was hidden

**Comment by alex-wilmer** [comment]
working for me on nightly

**Pull request merged**